docs/CONFIGURATIE.md

```diff
@@ -5,6 +5,13 @@ Bij de installatie van KISS worden er een groot aantal dingen al geconfigureerd.
 ## Configuratie van uw Identity Provider
 Bij de installatie van KISS regelt u de koppeling met uw OpenIDConnect Identity Provider. Daarnaast moet u in uw Identity Provider configureren dat gebruikers die in moeten kunnen loggen bij KISS in ieder geval een 'klantcontactmedewerker'-rol hebben. Een rol is in dit geval een claim van het type `role` of `roles` (beiden worden ondersteund). De waarde die correspondeert met een kiss-medewerker kunt u instellen tijdens de installatie. Standaard is dit `Klantcontactmedewerker`. Voor medewerkers die beheertaken op KISS uitvoeren, is een aparte rol ingeregeld. Ook de naam van deze rol kunt u instellen tijdens de installatie. Standaard is dit `Redacteur`. 
 
+### Claims uit uw Identity provider
+KISS gebruikt de claims uit uw Identity Provider om de gegevens van de ingelogde Klantcontactmedewerker toe te voegen aan de Contactmomenten en Contactverzoeken die vanuit KISS worden geregistreerd. Er is in JWT geen standaard claim voor voorletters of voorvoegsel (tussenvoegsel). KISS gebruikt daarom deze mapping:
+
+- Voorletters => given_name
+- Achternaam => family_name indien beschikbaar, anders name indien beschikbaar, anders http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name
+- Omdat voorvoegsel niet beschikbaar is, voegt KISS die bij de Achternaam.

```

Comment:
Dit is niet helemaal juist. Omdat er geen voorvoegsel-claim bestaat, doen we er niks mee. In Azure Active Directory kennen ze **ook** geen voorvoegsel en daarom stoppen we in ons KISS-AD **daar** het voorvoegsel bij de Achternaam bij het opvoeren van accounts.

Reply:
Ik heb de formulering aangepast.
@felixcicatt , is dit beter / acceptabel?